index.md

```diff
@@ -12,5 +12,5 @@ Open Horizon is a platform for managing the service software lifecycle of contai
 
 Edge computing brings computation and data storage closer to where data is created by people, places, and things.  Open Horizon simplifies the job of getting the right applications and machine learning onto the right compute devices and keeps those applications running and updated.  It also enables the autonomous management of more than 10,000 edge devices simultaneously – that’s 20 times as many endpoints as in traditional solutions.
 
-*   To install everything with a one-liner script, visit the [Quick Start](/quick-start/) page.
+*   To install everything with a one-liner script, visit the [Quick Start](./quick-start.md) page.
```

Comment:
The current link works as-is.   Is this change just to satisfy the Orphan link checker?

Reply:
Yes, think of it as an experiment.